flybywiresim/a32nx
# fix: correct cost index imported from SimBrief API

## Summary of Changes
I changed the simbrief flight plan import to multiply the cost index by 10, because in the mod's ATSU, the cost index ranges from 0-1000, whereas simbrief ranges from 0-100.

## Screenshots (if necessary)
### Simbrief flight plan for testing:
![Screenshot (69)](https://user-images.githubusercontent.com/2083713/104110304-567e9480-528b-11eb-9e6c-6822bc865176.png)
### Before
![Screenshot (68)](https://user-images.githubusercontent.com/2083713/104110308-5ed6cf80-528b-11eb-9e27-48dc02b5fcb3.jpg)

### After
![Screenshot (70)](https://user-images.githubusercontent.com/2083713/104110310-639b8380-528b-11eb-84b0-beebcefed70f.jpg)


## References





## Additional context



Discord username (if different from GitHub): mattsains#8270

## Testing instructions

1. Create a flight plan on simbrief for the A320. No settings are really important, but take a note of the cost index.
2. Start the sim with the A320 cold and dark. Power on the avionics
3. Click "Request" on the init page. The cost index can be viewed from the same page and should be a value between 0 and 1000.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
